chia/types/blockchain_format/program.py

```diff
@@ -72,6 +72,21 @@ def at(self, position: str) -> "Program":
                 raise ValueError(f"`at` got illegal character `{c}`. Only `f` & `r` allowed")
         return v
 
+    def replace(self, **kwargs) -> "Program":
```

Comment:
The API here is the best part of this function. If you want to replace the item "at" `ffr` you go `p.replace(ffr=new_value)`.

It may see weird for a casual observer, but for someone who's been deep in the weeds with clvm, it's really convenient.

Reply:
+1 to that